### PR TITLE
283 Show JOLA rows in post-inspection editor

### PR DESCRIPTION
**Pull request by danieldunderfelt**
Closes https://github.com/HSLdevcom/bultti/issues/283

Server PR: https://github.com/HSLdevcom/bultti/pull/285

Shows JOLA rows in a new box in the post-inspection editor. No actual sanctions or reports created yet.

For testing, HelB does have some results between January and May. Recommended to use a long inspection period to get some results.

### CODE REVIEW

File: src/text/fi.json [hunk 1]
```diff
@@ -240,6 +240,18 @@
   "postInspection_sanctionReason_UNIT_EQUIPMENT_MAX_AGE_VIOLATION": "Kohteen maksimi-iän ylitys",
   "postInspection_sanctionReason_EMISSION_CLASS_DEFICIENCY": "Suoritevaatimuksen alitus",
 
+  "postInspection_jolaPreview_heading": "JOLA esikatselu",
+  "postInspection_jolaPreview_description": "Näiden JOLA-rivien perusteella tehdään tähän jälkitarkastukseen sisältyvä kalustovikasanktiointi. Tämä on pelkkä esikatselu; kun tarkastus viedään sanktioitavaksi, lopulliset rivit haetaan uudestaan JOLA:sta.",
```
Comment (Divinare) [inline src/text/fi.json:244]: I would say:
"...lopulliset rivit haetaan JOLA:sta kun jälkitarkastus asetetaan hyväksyttäväksi."

Comment (Divinare) [reply src/text/fi.json:244]: This functionality should be quite clear for users, maybe put it behind a question mark (?)

**Review by Divinare — CHANGES_REQUESTED** [review]
Seems to work, saw one JOLA-sanction in the table when used February in postInspection, then encountered a cache bug - added new card to fix it: https://github.com/HSLdevcom/bultti/issues/286

Still would like to test with a longer period

File: src/text/fi.json (hunk 1 above)
Comment (danieldunderfelt) [reply src/text/fi.json:244]: Yes we can use the question mark! Texts written by me are mainly placeholders, feel free to change them.

**Review by Divinare — APPROVED** [review]

**Pull request merged**